### PR TITLE
A tool to detect openssl version

### PR DESCRIPTION
**Pull request by lacostej**
I mean this PR as discussion not necessarily as code to merge right away.

### CODE REVIEW

File: lib/fastlane_core/helper.rb [hunk 1]
```diff
@@ -102,6 +102,19 @@ def self.iterm?
       !!ENV["ITERM_SESSION_ID"]
     end
 
+    def self.openssl_version
+      return @openssl_version if @openssl_version
+
+      begin
+        output = `openssl version`
```
Comment (KrauseFx) [inline lib/fastlane_core/helper.rb]: Does this actually return the right version when you use `rvm` for example?

Comment (lacostej) [inline lib/fastlane_core/helper.rb]: Why woul `rvm` affect this ?

Comment (lacostej) [inline lib/fastlane_core/helper.rb]:
```
$ bundle exec ruby -e "require 'fastlane_core'; puts FastlaneCore::Helper.openssl_version"
1.0.2d
$ rvm list

rvm rubies

 * ruby-2.0.0-p645 [ x86_64 ]
   ruby-2.1.6 [ x86_64 ]
   ruby-2.1.7 [ x86_64 ]
   ruby-2.2.2 [ x86_64 ]
=> ruby-2.2.4 [ x86_64 ]
```

Comment (KrauseFx) [inline lib/fastlane_core/helper.rb]: Because rvm bundles its own version of OpenSSL AFAIK

Comment (lacostej) [inline lib/fastlane_core/helper.rb]: MMM I wasn't aware of this option. I always installed it uing brew.

http://stackoverflow.com/questions/14679878/openssl-error-got-when-install-ruby-2-0-0-via-rvm-and-rbenv

I will have to test that. Intuitively, it should work, as rvm modifies the PATH so the `openssl version` call should represent what rvm sees as well. But better to test more... I'll need some time to set this up.

**Comment by asfalcone** [comment]
Hey @lacostej I'm going to close this for now since it isn't going to be something we merge before the monorepo. Happy to continue the discussion if you want to though!

**Pull request closed without merging**